packages/rest/src/controllers/credentials/CredentialController.ts

```diff
@@ -157,6 +158,31 @@ export class CredentialController extends Controller {
     }
   }
 
+  /**
+   * Initiate a new credential exchange as issuer by creating a credential offer
+   * without specifying a connection id
+   *
+   * @param options
+   * @returns AgentMessage, CredentialExchangeRecord
+   */
+  @Example<CredentialExchangeRecordProps>(CredentialExchangeRecordExample)
+  @Post('/create-offer')
+  public async createOffer(
+    @Body() options: CreateOfferOptions,
+    @Res() internalServerError: TsoaResponse<500, { message: string }>
+  ) {
+    try {
+      const offer = await this.agent.credentials.createOffer(options)
+      await this.agent.oob.createInvitation({ messages: [offer?.message] })
```

Comment:
Ah wait, this shouldn't call create invitation. There's multiple ways to create the invitation, and also a legacy way. So what we've done is first call create offer endpoint, and you the. Manually need to call create (legacy-connection less) invitation endpoint

Reply:
Okay

Reply:
When you say endpoint, you mean the method, correct?